### PR TITLE
git-pr: add csr sub-command

### PR DESCRIPTION
**Pull request by edvbld**
Hi all,

please review this patch that adds the `git pr csr` sub-command that corresponds to the [`/csr`](https://wiki.openjdk.java.net/display/SKARA/Pull+Request+Commands#PullRequestCommands-/csr) pull request command. A contributor who prefers the command-line can now state that a pull request requires a [CSR](https://wiki.openjdk.java.net/display/csr) by running `git pr csr --needed`.

Testing:
- [x] Manual testing on Linux x64

Thanks,
Erik

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

### Reviewers
 * Jorn Vernee ([jvernee](@JornVernee) - **Reviewer**) ⚠️ Review applies to d11bdb543d6208f1fa8ea0bf7a9de37eb8f90b54

### Download
`$ git fetch https://git.openjdk.java.net/skara pull/702/head:pull/702`
`$ git checkout pull/702`

### CODE REVIEW

File: bots/pr/src/main/java/org/openjdk/skara/bots/pr/CSRCommand.java [hunk 1]
```diff
@@ -34,7 +34,7 @@
     private static final String CSR_LABEL = "csr";
 
     private static void showHelp(PrintWriter writer) {
-        writer.println("usage: `/csr [unneeded]`, requires that the issue the pull request links to an approved [CSR](https://wiki.openjdk.java.net/display/csr/Main) request.");
+        writer.println("usage: `/csr [needed|unneeded]`, requires that the issue the pull request links to an approved [CSR](https://wiki.openjdk.java.net/display/csr/Main) request.");
```
Comment (JornVernee) [inline bots/pr/src/main/java/org/openjdk/skara/bots/pr/CSRCommand.java:37]: Looks like a pre-existing typo:
```suggestion
        writer.println("usage: `/csr [needed|unneeded]`, requires that the issue the pull request links to, links to an approved [CSR](https://wiki.openjdk.java.net/display/csr/Main) request.");
```

File: bots/pr/src/test/java/org/openjdk/skara/bots/pr/CSRTests.java [hunk 2]
```diff
@@ -239,7 +249,7 @@ void showHelpMessageOnUnexpectedArg(TestInfo testInfo) throws IOException {
             TestBotRunner.runPeriodicItems(prBot);
 
             // Show help
-            assertLastCommentContains(pr, "usage: `/csr [unneeded]`, requires that the issue the pull request links " +
+            assertLastCommentContains(pr, "usage: `/csr [needed|unneeded]`, requires that the issue the pull request links " +
                                           "to an approved [CSR](https://wiki.openjdk.java.net/display/csr/Main) request.");
```
Comment (JornVernee) [inline bots/pr/src/test/java/org/openjdk/skara/bots/pr/CSRTests.java:253]: Same here
```suggestion
            assertLastCommentContains(pr, "usage: `/csr [needed|unneeded]`, requires that the issue the pull request links " +
                                          "to, links to an approved [CSR](https://wiki.openjdk.java.net/display/csr/Main) request.");
```

File: bots/pr/src/main/java/org/openjdk/skara/bots/pr/CSRCommand.java (hunk 1 above)
Comment (JornVernee) [reply bots/pr/src/main/java/org/openjdk/skara/bots/pr/CSRCommand.java:37]: Assuming the requirement is PR -(links to)-> Issue -(links to)-> CSR

Comment (edvbld) [reply bots/pr/src/main/java/org/openjdk/skara/bots/pr/CSRCommand.java]: Thanks for noticing this, I changed the usage message in a970e4ce2027a8f35d7e9dfffdc072e7c472b369. I used "... the issue the pull request refers to links to an approve" in order to not get to consecutive "links to" :smiley_cat: What do you think?

File: bots/pr/src/test/java/org/openjdk/skara/bots/pr/CSRTests.java (hunk 2 above)
Comment (edvbld) [reply bots/pr/src/test/java/org/openjdk/skara/bots/pr/CSRTests.java]: Thanks, see my other reply above :point_up:

File: bots/pr/src/main/java/org/openjdk/skara/bots/pr/CSRCommand.java (hunk 1 above)
Comment (JornVernee) [reply bots/pr/src/main/java/org/openjdk/skara/bots/pr/CSRCommand.java]: Looks good 👍

Comment (edvbld) [reply bots/pr/src/main/java/org/openjdk/skara/bots/pr/CSRCommand.java]: :+1:

**Pull request closed without merging**